### PR TITLE
v2017.1.x: docs: add missing devices

### PR DESCRIPTION
**Pull request by achterin**
This PR adds 2 devices to the docs that were recently added.
Devices are:

- GL Innovations GL-AR300M
- Ubiquiti UniFi AC Mesh
- TP-Link TL-WR940N v5
- TP-Link TL-WR1043N/ND

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -204,6 +204,7 @@ ar71xx-generic
   - UniFi AP Pro
   - UniFi AP Outdoor
   - UniFi AP Outdoor+
+  - UniFi AC Mesh [#ath10k]_
```
Comment (NeoRaider) [inline docs/index.rst]: The name and the ordering are inconsitent: It should be called UniFi AP AC Mesh and be ordered between the Lite and the Pro.

Comment (achterin) [reply docs/index.rst]: But that device has no "AP" in its Name. Its just called "UniFi AC Mesh". I just checked their productpage.
You are right though about the ordering. Will fix that when i get back home.

Comment (rotanid) [reply docs/index.rst]: well, you're partly correct, it is also named UAP-AC-M by the vendor...
but upstream OpenWrt doesn't mention the "AP" so we should change this in master branch and your naming should be ok

**Comment by achterin** [comment]
I've fixed the ordering and added 2 more devices, which are already supported.

**Comment by rotanid** [comment]
LGTM, merged

**Pull request merged**